dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -491,7 +493,7 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
 
 						{displayAvatarUrl ? (
 							<GridItem area="headline">
-								<div css={[avatarHeadlineWrapper, avatarUrl]}>
+								<div css={avatarHeadlineWrapper}>
```

Comment:
Why was a URL being passed to the `css` prop? 🤔

Reply:
Seems to have been a mistake when copying the styles over from `CommentLayout` where it's used to apply a minimum height when there's an avatar: `avatarUrl && minHeightWithAvatar`